src/services/AddressService.test.js

```diff
@@ -0,0 +1,58 @@
+import { getFullAddress } from './AddressService';
+
+describe('parseAddress', () => {
+    it('should return a friendly message when the address is empty', () => {
+        //arrange
+        const emptyAddress = {};
+
+        //act
+        const actual = getFullAddress(emptyAddress);
+
+        //assert
+        expect(actual).toEqual(
+            '1745 T Street Southeast1, Louisville, KY  40219'
```

Comment:
your test says that you will return a friendly message when the address is empty, but it looks like you are returning an actual address?